data_processors/pipeline/lambdas/dragen_wgs_qc.py

```diff
@@ -157,11 +161,29 @@ def handler(event, context) -> dict:
         sample_name=library_id
     )
 
+    # Get each of the engine parameters
+    subject_id = get_subject_id_from_libary_id(library_id)
+
+    # Get timestamp
+    timestamp = datetime.now().strftime("%Y-%m-%d__%H-%M-%S")
+
+    # Engine parameter subpath
+    engine_parameter_subpath = Path(subject_id) / \
```

Comment:
This path building and next line for building `engine_parameters` assignment could be a method (behaviour) of `WorkflowHelper` object. Or it can also be modelled as standalone / more specific `EngineParametersHelper(Helper)`.

Reply:
Created an EngineParametersHelper at https://github.com/umccr/data-portal-apis/blob/add-engine-parameters/data_processors/pipeline/domain/workflow.py#L58-L115

Reply:
Yup; this looks good. Nicely done, Alexi 👍